src/theme.js

```diff
@@ -46,13 +47,25 @@ const theme = createMuiTheme({
       letterSpacing: 'normal',
       fontSize: '0.6875rem',
     },
+    helperText: {
+      fontSize: '0.85rem',
+      lineHeight: '1.5em',
+    },
   },
   overrides: {
     MuiButton: {
       contained: {
         boxShadow: 'none',
       },
     },
+    MuiStepIcon: {
+      root: {
+        '&$completed': {
+          color: SecondaryMainColor,
+        },
+      },
+      completed: {},
```

Comment:
Do we need this?

Reply:
We want to use green for the completed state. As discussed in [trello ](https://trello.com/c/VlklzvPZ/73-placeholder-upfront-assessment-january-polish).

Reply:
Got it -- please remove unnecessary line `completed: {},`

Reply:
removed